src/write.jl

```diff
@@ -154,6 +155,7 @@ end
 # handle unknown schema case
 function write(::Nothing, rows, file::Union{String, IO};
     delim::Union{Char, String}=',',
+    recorddelim::Union{Char, String}='\n',
```

Comment:
I'd rather just call this `newline`

Reply:
Happy to oblige... but technically the record seperator doesn't have to be a line seperator at all.

For example, our literal matrices might have field delimiter `' '` and record delimiter `;` and no newlines.

Reply:
Maybe `newrow`?